docs/fxa-engineering/fxa-metrics.md

```diff
@@ -176,7 +176,7 @@ You don’t need to do this, it’s all set up to work automatically.
 # What other docs are there?
 * https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/metrics-events.md
 * https://github.com/mozilla/fxa/blob/main/packages/fxa-content-server/docs/client-metrics.md
-* https://github.com/mozilla/application-services/blob/master/docs/product-portal/accounts/metrics.md
+* https://github.com/mozilla/application-services/blob/main/docs/metrics.md
```

Comment:
This is a 404 too.  Looks like this should be https://github.com/mozilla/application-services/tree/main/docs/metrics